### DISCUSSION TITLE
Prefetch is not working fine on mobile devices

### DISCUSSION THREAD

**Discussion by sheikhfahad67** [vercel/next.js#56952] [App Router]
Hi,

Hope you all doing great. I need your help in regard to the "Link" component prefetch mechanism.

I applied it on my all routes to make them prefetch, on desktop browsers, it is working fine but when I open it on the mobile phone browsers it will not work the same as on the desktop.

Can anyone let me know why this happened?

Thanks!

**Comment by FlyingTurkman** [comment]
Which next.js version?

**Reply by sheikhfahad67** [reply]
It's v13.5

**Reply by FlyingTurkman** [reply]
Its deprecated. https://nextjs.org/docs/messages/prefetch-true-deprecated

**Comment by yasseralsaidi** [comment]
I had to use prefetch={false} in every link for it to work properly. Is there any fix?
